src/braft/fsm_caller.cpp

```diff
@@ -78,10 +78,13 @@ int FSMCaller::run(void* meta, bthread::TaskIterator<ApplyTask>& iter) {
                 max_committed_index = -1;
                 g_commit_tasks_batch_counter << counter;
                 counter = 0;
-                batch_size = FLAGS_raft_fsm_caller_commit_batch;
```

Comment:
This line is used to determine if the batch size is changed manually.

Reply:
It seems that FLAGS_raft_fsm_caller_commit_batch not dynamically revised by using "FLAGS_raft_fsm_caller_commit_batch=" or "SetCommandLineOption" in the codes, and batch_size has got its value in line 67

Reply:
gflags registered with BRPC_VALIDATE_GFLAG can be dynamically changed.